Update sdk-platform and v4-develop in urls

piiq opened the pull request.

hjoaquim commented on website/pypi.md:
````diff
@@ -22,7 +22,7 @@ The command below provides access to the core functionalities behind the [OpenBB
 ```
 
-If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/sdk/installation).
+If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/platform/installation).
````
Those toolkits are not yet supported on the Platform, should we still point this section to the platform installation procedure?

piiq replied:
good catch. maybe just pointing to the docs root?

piiq replied:
reverted

piiq commented:
Thanks for proof-reading this @hjoaquim. I've reverted the changes that were wrongfully introduced for v3 documentation and changed the base branch of this PR to be the docs PR. This PR depends on the docs PR, so if it's merged the links won't be broken.

piiq commented on website/pypi.md:
````diff
@@ -32,7 +32,7 @@ Access our fully fledged financial SDK with a single line of python code.
 ```
 
-Everything you need to use the OpenBB SDK can be found on our [Official Documentation](https://docs.openbb.co/sdk).
+Everything you need to use the OpenBB SDK can be found on our [Official Documentation](https://docs.openbb.co/platform).
````
reverted
